ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__arm64-nightly.yaml

```diff
@@ -95,6 +95,23 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-baselinecaps-none
+- as: aws-ipi-baselinecaps-v413-p2-f14
```

Comment:
I guess we also need a `destructive` testing for the install.

Reply:
Done